envconfig/config.go

```diff
@@ -237,6 +244,8 @@ func AsMap() map[string]EnvVar {
 		"OLLAMA_FLASH_ATTENTION":   {"OLLAMA_FLASH_ATTENTION", FlashAttention(), "Enabled flash attention"},
 		"OLLAMA_KV_CACHE_TYPE":     {"OLLAMA_KV_CACHE_TYPE", KvCacheType(), "Quantization type for the K/V cache (default: f16)"},
 		"OLLAMA_GPU_OVERHEAD":      {"OLLAMA_GPU_OVERHEAD", GpuOverhead(), "Reserve a portion of VRAM per GPU (bytes)"},
+		"OLLAMA_DRAFT_GPU_DEVS":    {"OLLAMA_DRAFT_GPU_DEVS", DraftGpuDevs(), "Comma-separated list of devices to use for offloading the draft model (default: same GPU devices as available for target model)"},
+		"OLLAMA_DRAFT_GPU_LAYERS":  {"OLLAMA_DRAFT_GPU_LAYERS", DraftGpuLayers(), "Number of layers to store in VRAM for the draft model (default: 99)"},
```

Comment:
Ollama automatically manages model layer placement for the main model, I think the same should be true for the draft model.

Reply:
Thanks, I agree! At first I held back regarding automatic model layer placement, because a lot needs to be touched and I was doubtful that maintainers would accept such a large change. Anyway, see the fix regarding a first attempt. FIX: https://github.com/ollama/ollama/pull/8134/commits/59e036b6b0e3b2674c27c7d57aae0f7d529c0f31